<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsActionMenu</class>
 <widget class="QDialog" name="ContactsActionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="5" column="0">
      <widget class="QPushButton" name="btDelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/delete.png</normaloff>
         <disabledoff>:/icons/delete-disbl.png</disabledoff>
         <disabledon>:/icons/delete-disbl.png</disabledon>:/icons/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/add-contact.png</normaloff>
         <disabledoff>:/icons/add-contact-disbl.png</disabledoff>
         <disabledon>:/icons/add-contact-disbl.png</disabledon>:/icons/add-contact.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btEdit">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/edit.png</normaloff>
         <disabledoff>:/icons/edit-disbl.png</disabledoff>
         <disabledon>:/icons/edit-disbl.png</disabledon>:/icons/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btCall">
       <property name="text">
        <string>Call</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/call.png</normaloff>
         <disabledoff>:/icons/call-disbl.png</disabledoff>
         <disabledon>:/icons/call-disbl.png</disabledon>:/icons/call.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>81</width>
         <height>81</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btMsg">
       <property name="text">
        <string>Message</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/messagess.png</normaloff>:/icons/messagess.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="taki.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btCall</sender>
   <signal>released()</signal>
   <receiver>ContactsActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAdd</sender>
   <signal>released()</signal>
   <receiver>ContactsActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEdit</sender>
   <signal>released()</signal>
   <receiver>ContactsActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDelete</sender>
   <signal>released()</signal>
   <receiver>ContactsActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btMsg</sender>
   <signal>released()</signal>
   <receiver>ContactsActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
